oat-sa/extension-tao-testqti
# Fix/tao 4279 export error report

Fixed the test part of TAO-4279. Exception are properly caught and output as standard report.
To test this:
1. Create a test subclass
2. Create a test in it and add an item with no content (just created in the item extension but with no content saved from authoring tool)
3. Launch the export and check the result
